packages/create-cloudflare/src/frameworks/index.ts

```diff
@@ -3,20 +3,17 @@ import { dim } from "@cloudflare/cli/colors";
 import { quoteShellArgs, runCommand } from "helpers/command";
 import { detectPackageManager } from "helpers/packageManagers";
 import { isInsideGitRepo } from "../git";
-import clisPackageJson from "./package.json";
+import frameworksPackageJson from "./package.json";
 import type { C3Context } from "types";
 
 export const getFrameworkCli = (ctx: C3Context, withVersion = true) => {
 	if (!ctx.template) {
 		return crash("Framework not specified.");
 	}
 
-	const framework = ctx.template
-		.id as keyof typeof clisPackageJson.frameworkCliMap;
-	const frameworkCli = clisPackageJson.frameworkCliMap[
-		framework
-	] as keyof typeof clisPackageJson.dependencies;
-	const version = clisPackageJson.dependencies[frameworkCli];
+	const frameworkCli = ctx.template
+		.frameworkCli as keyof typeof frameworksPackageJson.dependencies;
```

Comment:
The source of truth for mapping between frameworks and their CLI has moved to the `c3.ts` template so that is all kept in one place, allowing us to remove that map from the frameworks/package.json